i-dot-ai/evaluation-registry
# feature/create-cognito-user

A new management command that allows admins to create new users in cognito.

Out of scope:
* creating administrators, this will have to be a manual task
* a UI 

to test:
1. delete yourself from https://eu-west-2.console.aws.amazon.com/cognito/v2/idp/user-pools/eu-west-2_GB9DvxhLp/users?region=eu-west-2
2. logout 
3. attempt to login, you should fail
4. deploy this branch
5. run this management command 
https://eu-west-2.console.aws.amazon.com/ec2/home?region=eu-west-2#Instances:instanceState=running;v=3;$case=tags:true%5C,client:false;$regex=tags:false%5C,client:false select a dev instance and connect then run
```commandLine
eval $(/opt/elasticbeanstalk/bin/get-config environment | jq -r 'to_entries | .[] | "export \(.key)=\(.value)"' )
source /var/app/venv/staging-LQM1lest/bin/activate
python /var/app/current/manage.py create_users your.email@cabinetoffice.gov.uk
```
7. login, you should succeed
